File: SoloLearn Codes/Izzy the Iguana/izzy_the_iguana.cpp
#include <iostream>
#include <map>
#include <string>
#include <sstream>
using namespace std;

int main() {
    map<string, int> map = {
        {"Mango", 9},
        {"Carrot", 4},
        {"Lettuce", 5},
        {"Cheeseburger", 0}
    };

    string str;
    getline(cin, str);
    stringstream ss(str);
    int p = 0;
    while (getline(ss, str, ' ')) {
        if (map.find(str) != map.end()) p += map[str];
    }
    cout << ((p >= 10) ? "Come on Down!" : "Time to wait");

    return 0;
}

File: SoloLearn Codes/Izzy the Iguana/izzy_the_iguana.rb
hash = { "Mango"=> 9, "Carrot"=> 4, "Lettuce"=> 5, "Cheeseburger"=> 0 }
puts(gets.chomp.split(" ").map{ |x| hash[x] || 0 }.sum >= 10 ? "Come on Down!" : "Time to wait")